shared/headers/create-address.yml

```diff
@@ -0,0 +1,7 @@
+send_date:
+  description:
+    A timestamp of the date on which a mailpiece will be dispatched
+    for production. Beyond this date, the mailpiece can't be cancelled.
+  schema:
+    type: string
```

Comment:
We can use `format: date-time` here as well as string. https://swagger.io/docs/specification/data-models/data-types/#string